### DISCUSSION TITLE
Scheduled restarts

### DISCUSSION THREAD

**Discussion by Devidian** [mbround18/valheim-docker#273] [Ideas]
Hi there, 

my server is running since the last update and i noticed that it starts to begin lagging more and more each evening we stream. Since the auto-update does not stop/start the server it might be good to have a 
`SCHEDULED_RESTART=1/0` and  `SCHEDULED_RESTART_SCHEDULE=0 5 * * *` something. 

Best regards
Devidian

**Comment by mbround18** [comment]
That's not a bad idea :P I'll look into it

**Reply by Phoenix-Game** [reply]
Have you done anything about that? Is there possible to make scheduled restarts or not yet?

**Reply by mbround18** [reply]
@Phoenix-Game i can, gimme a day or so. tbh got busy with work nd moving.

**Reply by mbround18** [reply]
Aight got the code up in https://github.com/mbround18/valheim-docker/pull/588 will do some testing on it later and get it merged.

**Reply by Phoenix-Game** [reply]
Alright, thanks.

**Reply by mbround18** [reply]
@Phoenix-Game ive tried it locally nd it looks like it works great :) going to get it vetted by the other devs but you are welcome to test it out with 

```
mbround18/valheim:sha-89f3dfe
```

**Reply by Phoenix-Game** [reply]
@mbround18 It works just perfcet, thanks, would like to see this feature in the latest version. 👍

**Reply by mbround18** [reply]
@Phoenix-Game this feature is available on V1.6.0
